rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -306,10 +306,38 @@
       "description": "Document is harvested or not, will disable record edition or similar.",
       "type": "boolean"
     },
-    "fiction": {
-      "title": "Fiction",
-      "description": "Document is fiction.",
-      "type": "boolean",
+    "fiction_statement": {
+      "title": "Fiction statement",
+      "description": "Fiction statement.",
+      "type": "string",
+      "default": "unspecified",
+      "enum": [
+        "fiction",
+        "non_fiction",
+        "unspecified"
+      ],
+      "widget": {
+        "formlyConfig": {
+          "type": "selectWithSort",
+          "props": {
+            "itemCssClass": "col-lg-6",
+            "options": [
+              {
+                "label": "fiction",
+                "value": "fiction"
+              },
+              {
+                "label": "non_fiction",
+                "value": "non_fiction"
+              },
+              {
+                "label": "unspecified",
+                "value": "unspecified"
+              }
+            ]
+          }
+        }
+      },
       "form": {
         "hide": true
```

Comment:
You can delete this. With a default value, the field can only be hidden manually.